services/__tests__/static.test.js

```diff
@@ -39,6 +39,100 @@ describe('getStaticImage', () => {
     });
   });
 
+  test('bbox', () => {
+    service.getStaticImage({
+      ownerId: 'mapbox',
+      styleId: 'streets-v10',
+      width: 200,
+      height: 300,
+      position: {
+        bbox: [-77.04, 38.8, -77.02, 38.91]
+      }
+    });
+    expect(tu.requestConfig(service)).toEqual({
+      method: 'GET',
+      path:
+        '/styles/v1/:ownerId/:styleId/static/[-77.04,38.8,-77.02,38.91]/200x300',
+      query: {},
+      params: { ownerId: 'mapbox', styleId: 'streets-v10' },
+      encoding: 'binary'
+    });
+  });
+
+  test('padding', () => {
```

Comment:
[request] this is great, can we also add tests for other forms of padding? (i.e. 1, 3, & 4 comma delimited values)